src/main/res/drawable-ldrtl/message_bubble_background_sent_alone_with_border.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:top="2px"
+        android:bottom="2px">
+        <shape android:shape="rectangle">
+            <stroke android:width="1dp" android:color="#55888888" />
+            <corners
+                android:topLeftRadius="@dimen/message_corner_radius"
+                android:topRightRadius="@dimen/message_corner_radius"
+                android:bottomRightRadius="@dimen/message_corner_radius" />
+            <!-- read issue: https://github.com/deltachat/deltachat-android/issues/480 before introducing an attr here. -->
+            <solid android:color="?android:attr/windowBackground" />
```

Comment:
why not use `?attr/conversation_item_incoming_bubble_color` as the previous code?